sound/soc/sof/loader.c

```diff
@@ -192,6 +192,29 @@ static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int ext_man_get_config_data(struct snd_sof_dev *sdev,
```

Comment:
@fredoh9 the code looks good now. just a comment on the commit message: could you pleas add that a new member is added to struct snd_sof_dev to store the FW clock config information and that this will be used later to perform platform-specific operations in the post_fw_run op?

Reply:
It is not right to say that this will be used to initialize the FW as you get this info after FW boot.